api.py

```diff
@@ -47,32 +50,24 @@ def create_ffmpeg_stream(video_path:str, video_type:State, loop=False):
     )
     
 def handle_interlude():
-    interlude_exists = args.interlude is None or not os.path.exists(args.interlude)
+    interlude_exists = args.interlude is not None or os.path.exists(args.interlude)
     while True:
         interlude_lock.acquire()
         # Check if interlude video file exists:
         if interlude_exists:
-            process_dict[State.INTERLUDE] = None
-        else:
             create_ffmpeg_stream(args.interlude, State.INTERLUDE, True)
+        else:
+            process_dict[State.INTERLUDE] = None
 
```

Comment:
for this function lets just do:
```py
def handle_interlude():
    while True:
        interlude_lock.acquire()
        # Check if interlude video file exists:
        create_ffmpeg_stream(args.interlude, State.INTERLUDE, True)
```

`handle_interlude` should assume that the interlude file exists